docs/tutorials/using-typeorm.md

```diff
@@ -5,30 +5,32 @@ description: Learn how to create and register TypeORM entities and perform trans
 ---
 
 ## TypeORM Overview
-[TypeORM](https://typeorm.io) is an ORM for TypeScript.  It is based on the idea of creating [`Entity`](https://typeorm.io/entities) classes to represent each database table, with the persistent and join key fields marked with [decorators](https://typeorm.io/decorator-reference).  Once entity classes are defined, TypeORM provides methods for storing, updating, and querying the entities via the [`EntityManager`](https://typeorm.io/working-with-entity-manager).  TypeORM can also be used to create and maintain the database schema.
+[TypeORM](https://typeorm.io) is a popular TypeScript ORM.
+DBOS supports it as an alternative to [Knex](https://knexjs.org/) for transactional data management.
```

Comment:
I think it's good to put a pointer to entity manager here. It clarifies what we use in TypeORM for transactions.

Reply:
Restored the original text